app/controllers/transition_defence_requests_controller.rb

```diff
@@ -34,6 +34,7 @@ def transition_params
       defence_request: defence_request,
       transition_to: requested_transition,
       user: current_user,
+      auth_token: session[:user_token]
```

Comment:
Good point. I do think `auth_token` lives separately so we can ensure it is fresh, at least from looking at the implementation of the check for current user. We could slightly change the strategy builder to handle this, but if we have no additional information at this point, we might as well keep the way you've implemented it.